src/SpeechRecognition.js

```diff
@@ -40,7 +40,7 @@ export default function SpeechRecognition(options) {
 
       componentWillMount() {
         if (recognition) {
-          recognition.continuous = true
+          recognition.continuous = recognition.continuous = !options.continuous ? false : true
```

Comment:
Redundant assignment and changes the default to `false`. To preserve the default of `true` and simplify the expression, I suggest the following:

```suggestion
          recognition.continuous = options.continuous !== false
```